apps/etterlatte-beregning/src/main/kotlin/beregning/Beregning.kt

```diff
@@ -23,5 +25,14 @@ data class Beregning(
             beregningsperioder = beregningsperioder,
             beregnetDato = beregnetDato,
             grunnlagMetadata = grunnlagMetadata,
+            manueltBeregning = manueltBeregning.toDTO(),
         )
 }
+
+data class ManueltBeregning(
```

Comment:
Burde nok være `ManuellBeregning` hvis vi skal beholde den ordlyden.

Lurer på om vi egentlig bør kalle det for `OverstyrBeregning`, som er begrepet som vi benytter i Pesys også.

Reply:
Helt OK å omdøpe